docs/index.md

```diff
@@ -25,6 +25,7 @@ In this introductory workshop, you'll learn how to use **RHODS** to recognize li
 
 And all of this without having to install anything on your own computer, thanks to **RHODS**!
 
+<!---
```

Comment:
Why do you mask the whole section?

Reply:
good Q: I guess probably not, in the long term. I think by Thursday we will have something back in here - I just took it out for now incase we forget!